### PR TITLE
rocket landing problem solved using pid

### PR DESCRIPTION
**Pull request by NikhilSuresh24**
I got a solution to the rocket landing problem using pid by tuning constants.

### CODE REVIEW

**Comment by Codadillo** [comment]
Honestly I have to agree with Casey on this one. Needs a complete rewrite in rust. Big L.

**Comment by Codadillo** [comment]
I am considering closing this entirely. Complete waste of time.

File: rocket_landing.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function controlFunction(block)
+{
+  return 5*Math.sin(10*block.T);
```
Comment (CaseyManning) [inline rocket_landing.js]: this line is too long.

File: rocket_landing.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function controlFunction(block)
```
Comment (Codadillo) [inline rocket_landing.js]: Horrible start. Simply terrifying how someone could be so bad at coding.

**Pull request closed without merging**